config.js

```diff
@@ -572,6 +575,18 @@ var config = {
     //     customUrl: ''
     // },
 
+    // Configs for the lobby screen.
+    // lobby {
+    //     // Hides lobby button. Replaces `hideLobbyButton`.
+    //     hideLobbyButton: false,
```

Comment:
Hum, shouldn't this go into some (new!) "securityUi" section perhaps? I'm sorry.